Exercise_A/index.html

```diff
@@ -10,8 +10,30 @@
       name="viewport"
       content="width=device-width, initial-scale=1"
     />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
   </head>
   <body>
-    <!-- CODE HERE -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <img src='./assets/bootstrap.png' alt="" width="60" height="60">
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">

```

Comment:
Nice job implementing this bootstrap website, the only thing that doesnt match was the navigation bar links.
To remove the underlines in boostrap:
'text-decoration-none' within the class element